src/segmentmytif/utils/datasets.py

```diff
@@ -26,13 +29,23 @@ def image_path_to_mask_path(image_path: Path) -> Path:
             self.limit = len(self.images)
 
     def __getitem__(self, index):
-        img = transforms.ToTensor()(Image.open(self.images[index]).convert("L"))
+        img_arr = read_geotiff(self.images[index]).data
```

Comment:
I am not sure where `MonochromeFlairDataset` will be called. But if you are fine with a default dask setup, I think we can skip the dask setup here

Reply:
That is fine indeed, it was only used to prepare the training dataset for the feature extractor. It could very well be that it will never be run again.